Server/src/routes/issueToken.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { CommunicationUserToken, TokenScope } from '@azure/communication-identity';
+import * as express from 'express';
+import { createUserAndToken } from '../lib/identityClient';
+
+const router = express.Router();
+
+/**
+ * handleUserTokenRequest will return a default scoped token if no scopes are provided.
+ * @param requestedScope [optional] string from the request, this should be a comma seperated list of scopes.
+ */
+const handleUserTokenRequest = async (requestedScope?: string): Promise<CommunicationUserToken> => {
+  const scopes: TokenScope[] = requestedScope ? (requestedScope.split(',') as TokenScope[]) : ['chat', 'voip'];
```

Comment:
might be confusing that the chat sample is minting a token with voip here

Reply:
created an ADO item
https://skype.visualstudio.com/SPOOL/_workitems/edit/2691358